tox.ini

```diff
@@ -50,21 +53,24 @@ commands =
 basepython = python3.5
 deps =
     nose >= 1.2.1
-    rednose
+    pytest
```

Comment:
I understand why we're overriding the `deps` from `testenv` in `testenv:pypy` (see the comment), but why do we have to override them in the `nodeps` environments? @alvations @stevenbird come to think of it I'm not sure I understand what the `nodeps` environments are for :)

Reply:
there's an explanation here `web/dev/local_testing.rst` -- it seems like these environments were intended to have test requirements but not much beyond that? i can't speak to how often they're used at the moment though

Reply:
Ah, ok I understand what that's about then. We'll deal with that later then :)